src/Uno.Toolkit.UI/Behaviors/FlipViewExtensions.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.Logging;
+using Windows.System;
+using Windows.UI.ViewManagement;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Input;
+#endif
+
+namespace Uno.Toolkit.UI;
+
+public static class FlipViewExtensions
+{
+	public static DependencyProperty NextProperty { get; } =
+		DependencyProperty.RegisterAttached("Next", typeof(FlipView), typeof(FlipViewExtensions), new PropertyMetadata(null, OnNextChanged));
+
+	static void OnNextChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+	{
+		if (d is not ButtonBase btn || e.NewValue is null)
+			return;
+
+		var flipView = (FlipView)e.NewValue;
+
+		btn.Click -= OnBtnClick;
+		btn.Click += OnBtnClick;
```

Comment:
We can accomplish this using IDisposables, similar to [here](https://github.com/unoplatform/uno.toolkit.ui/blob/62914f3607c877e5ad0589c5b17d045c10527119/src/Uno.Toolkit.UI/Behaviors/TabBarSelectorBehaviorState.Uwp.cs#L31-L38)

So we can ensure the event subscription is dropped when setting the Disposable to null and then hook up a new subscription

Reply:
Instead of adding the Disposable, can I unsubscribe during the Unloaded event? Didn't find a good place to null the Disposable here, it always clean the event too early